src/systems/tracked_vehicle/TrackedVehicle.hh

```diff
@@ -0,0 +1,154 @@
+inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace systems
+{
+  // Forward declaration
+  class TrackedVehiclePrivate;
+
+  /// \brief Tracked vehicle controller which can be attached to a model
+  /// with any number of left and right tracks. The system should be attached
+  /// to a model. Each track has to have a TrackController system configured and
+  /// running.
+  ///
+  /// So far, this system only supports tracks that are parallel along a common
+  /// axis (other designs are possible, but not implemented).
+  ///
+  /// # Examples
+  ///
+  /// See example usage in world example/tracked_vehicle_simple.sdf .
+  ///
+  /// # System Parameters
+  ///
+  /// `<left_track>`: Configuration of a left track link. This element can
+  /// appear multiple times, and must appear at least once.
+  ///
+  /// `<right_track>`: Configuration of a right track link. This element can
+  /// appear multiple times, and must appear at least once.
+  ///
+  /// `<left_track>`, `<right_track>` subelements:
+  /// - `<link>`: The link representing the track.
+  /// - `<velocity_topic>`: The topic on which the track accepts velocity
+  ///     commands (defaults to `/model/${model}/link/${link}/track_cmd_vel`).
+  /// - `<center_of_rotation_topic>`: The topic on which the track accepts
+  ///     center of rotation commands (defaults to
+  ///     `/model/${model}/link/${link}/track_cmd_center_of_rotation`).
+  ///
+  /// `<tracks_separation>`: Distance between tracks, in meters.
+  ///
+  /// `<track_height>`: Height of the tracks, in meters (used for computing
+  /// odometry).
+  ///
+  /// `<steering_efficiency>`: Initial steering efficiency. Defaults to 0.5.
+  ///
+  /// `<debug>` If 1, the system will output debugging info and visualizations.
+  ///
+  /// `<linear_velocity>`: Limiter of linear velocity of the vehicle. Please
+  /// note that the tracks can each have their own speed limitations.
+  /// - `<min_velocity>`/`<max_velocity>` Min/max velocity of the vehicle (m/s).
+  /// - `<min_acceleration>`/`<max_acceleration>` Min/max acceleration of the
+  ///   vehicle (m/s^2).
+  /// - `<min_jerk>`/`<max_jerk>` Min/max jerk of the vehicle (m/s^3).
+  ///
+  /// `<angular_velocity>`: Limiter of angular velocity of the vehicle. Please
+  /// note that the tracks can each have their own speed limitations.
+  /// - `<min_velocity>`/`<max_velocity>` Min/max velocity of the vehicle
+  ///   (rad/s).
+  /// - `<min_acceleration>`/`<max_acceleration>` Min/max acceleration of the
+  ///   vehicle (rad/s^2).
+  /// - `<min_jerk>`/`<max_jerk>` Min/max jerk of the vehicle (rad/s^3).
+  ///
+  /// `<odom_publish_frequency>`: Odometry publication frequency. This
+  /// element is optional, and the default value is 50Hz.
+  ///
+  /// `<topic>`: Custom topic that this system will subscribe to in order to
+  /// receive command velocity messages. This element if optional, and the
+  /// default value is `/model/{name_of_model}/cmd_vel`.
```

Comment:
```suggestion
  /// receive command velocity messages. This element is optional, and the
  /// default value is `/model/{model_name}/cmd_vel`.
```

Or you can keep it as `{name_of_model}` but change the others for consistency

Reply:
fixed